openedx/features/enterprise_support/api.py

```diff
@@ -358,28 +361,63 @@ def get_enterprise_customer_cache_key(uuid, username=settings.ENTERPRISE_SERVICE
     )
 
 
-@enterprise_is_enabled()
 def cache_enterprise(enterprise_customer):
-    """Cache this customer's data."""
+    """Add this customer's data to the Django cache."""
     cache_key = get_enterprise_customer_cache_key(enterprise_customer['uuid'])
     cache.set(cache_key, enterprise_customer, settings.ENTERPRISE_API_CACHE_TIMEOUT)
 
 
-@enterprise_is_enabled()
-def enterprise_customer_from_cache(request=None, uuid=None):
-    """Check all available caches for Enterprise Customer data."""
-    enterprise_customer = None
+def enterprise_customer_from_cache(uuid):
```

Comment:
Removing the `enterprise_is_enabled` check, because it's innocuous to check the cache for a key, and presumably if any caller of this cares, that caller will do the check.